docs/src/main/asciidoc/dev-services.adoc

```diff
@@ -0,0 +1,75 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Dev Services Overview
+:extension-status: preview
+
+include::./attributes.adoc[]
+
+Quarkus supports the automatic provisioning of unconfigured services in development and test mode. We refer to this capability
+as Dev Services. From a developers perspective this means that if you include an extension and don't configure it then
+Quarkus will automatically start the relevant service (usually using Test Containers behind the scenes) and wire up your
+application to use this service.
+
+All this functionality is part of the Quarkus `deployment` modules, so does not affect the production application in any
+way.
+
+This page lists all the Dev Services that Quarkus supports.
+
+NOTE: In order to use Dev Services you will generally need a working docker environment (remote environments are supported).
+If you don't have docker installed you will need to configure your services normally.
+
+== Amqp
+
+Amqp will be enabled by adding `quarkus-smallrye-reactive-messaging-amqp` to your application.
+
+include::{generated-dir}/config/quarkus-smallrye-reactivemessaging-amqp-config-group-amqp-dev-services-build-time-config.adoc[opts=optional, leveloffset=+1]
```

Comment:
We have this page: https://quarkus.io/guides/amqp-dev-services . Apparently, it's not in the index, I will add it.